agent/js/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  ignorePatterns: ["dist"],
+  extends: "standard-with-typescript",
```

Comment:
To prevent collision between prettier and eslint
```suggestion
  extends: ["standard-with-typescript", "prettier"],
```

Reply:
Now they match, we are in unison.